### ISSUE TITLE
Separate the serve mode and the build mode

### ISSUE THREAD

**Issue by ByunghoJeon** [apenwarr/mkdocs-exclude#7]
Thanks for this plugin. It's helpful for me.
I found this plugin to solve slow performance serve mode. It's works nicely. 
Many files degrade serve mode performance.
It would be nice if I could separate the serve mode and the build mode.

Thanks.

**Comment by axgkl** [comment]
yes! was hoping this plugin solves a problem I have in serve mode:
I have a preprocessor evaluating fenced code stuff into the final markdown. Files with such stanzas have their own ending, so would be easy to match. Problem is every time I edit such files the livereloader rebuilds before the preprocessor is done creating the final md.
But I guess making the livereloader ignore changes in such files is out of scope of this plugin, right?
